src/main/java/net/minecraftforge/client/event/AddFramePassEvent.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+package net.minecraftforge.client.event;
+
+import com.mojang.blaze3d.framegraph.FrameGraphBuilder;
+import com.mojang.blaze3d.framegraph.FramePass;
+import net.minecraft.client.renderer.LevelTargetBundle;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.eventbus.api.Event;
+
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * Fired after all vanilla frame passes are added into the pass list.
+ * 
+ * <p>This event is not {@linkplain Cancelable cancellable}, and does not {@linkplain HasResult have a result}.</p>
+ *
+ * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS main Forge event bus},
+ * only on the {@linkplain LogicalSide#CLIENT logical client}.
+ */
```

Comment:
If you aren't going to use imports, you need to use fully-qualified class names for them to be linked in the JavaDocs. It's important not to have dead links in JavaDocs.

```suggestion
/**
 * Fired after all vanilla frame passes are added into the pass list.
 * 
 * <p>This event is not {@linkplain net.minecraftforge.eventbus.api.Cancelable cancellable}, and does not
 * {@linkplain HasResult have a result}.</p>
 *
 * <p>This event is fired on the {@linkplain net.minecraftforge.common.MinecraftForge#EVENT_BUS main Forge event bus},
 * only on the {@linkplain net.minecraftforge.fml.LogicalSide#CLIENT logical client}.
 */
```